back/src/Guide/Api/Migration.hs

```diff
@@ -0,0 +1,131 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Guide.Api.Migration where
+
+import Imports
+
+import Hasql.Session (Session)
+-- import Hasql.Statement (Statement(..))
+import Hasql.Connection (Connection, Settings)
+
+import qualified Hasql.Session as HS
+-- import qualified Hasql.Decoders as HD
+-- import qualified Hasql.Encoders as HE
+import qualified Data.ByteString as B
+import qualified Hasql.Connection as HC
+
+
+main :: IO ()
+main = do
+  conn <- connection
+  dbCreate conn
+
+connection :: IO Connection
+connection = do
+  eiConnection <- HC.acquire connectionSettings
+  let conn = either (error . show) id eiConnection
+  pure conn
+
+connectionSettings :: Settings
+connectionSettings = HC.settings "localhost" 5432 "postgres" "3" "guide"
+
+dbCreate :: Connection -> IO ()
+dbCreate conn = do
+  result <- mapM (\s -> HS.run s conn)
+    [createTypeProcons, createDbCategories, createDbItems, createDbTraits, createDbUsers, createDbEdits]
+  mapM_ (either print pure) result
+
+createTypeProcons :: Session ()
+createTypeProcons = HS.sql "CREATE TYPE trait_type AS ENUM ('pro', 'con');"
+
+createDbTraits :: Session ()
+createDbTraits = HS.sql $ B.intercalate " "
+  [ "CREATE TABLE traits ("
+  ,   "uid text PRIMARY KEY,"           -- Unique trait ID
```

Comment:
You can use http://hackage.haskell.org/package/neat-interpolation-0.3.2.4/docs/NeatInterpolation.html here together with some helper function like `sqlText`. Postgres supports Haskell-style comments so you don't need to take special care of them.